eos/b-decays/b-to-psd-psd-l-nu-impl.hh

```diff
@@ -0,0 +1,246 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2025 Florian Herren
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef EOS_GUARD_EOS_B_DECAYS_B_TO_PSD_PSD_L_NU_IMPL_HH
+#define EOS_GUARD_EOS_B_DECAYS_B_TO_PSD_PSD_L_NU_IMPL_HH 1
+
+#include <eos/observable.hh>
+#include <eos/b-decays/b-to-psd-psd-l-nu.hh>
+#include <eos/maths/angular-integrals.hh>
+#include <eos/maths/complex.hh>
+#include <eos/maths/power-of.hh>
+
+#include <array>
+#include <iostream>
+
+namespace eos
+{
+    namespace b_to_psd_psd_l_nu
+    {
+        struct Amplitudes
+        {
+            std::array<complex<double>, 4> f_perp;
+            std::array<complex<double>, 4> f_para;
+            std::array<complex<double>, 4> f_long;
+            std::array<complex<double>, 4> f_time;
+            double q2;
+            double betaL;
+            double betaPi;
+            double pref;
+        };
+
+        class AngularObservables
+        {
+            private:
+                std::array<std::array<double, 5>, 9> _m;
```

Comment:
Can we extend this name, to avoid accidental conflicts with the loop variable ``m``?